url_test.go

```diff
@@ -134,3 +134,37 @@ func equalStringSlice(a, b []string) bool {
 
 	return true
 }
+
+func TestIsLive(t *testing.T) {
+	var testValues = []struct {
+		Input string
+		Want  bool
+	}{
+		{
+			Input: "https://yagizdegirmenci.com",
+			Want:  true,
+		},
+		{
+			Input: "https://golang.org",
+			Want:  true,
+		},
+		{
+			Input: "https://xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.com",
+			Want:  false,
+		},
+		{
+			Input: "https://.com",
+			Want:  false,
+		},
+	}
+
+	for _, testValue := range testValues {
+		u, _ := NewURL(testValue.Input)
```

Comment:
Checking error would be fine here. For example, "https://.com" should returned an error becasue that's not a valid URL.

Reply:
Thanks, actually, I'm decided to change test cases because adding an error check here will go beyond the purpose of this test case.